redirects-list.json

```
@@ -73,6 +73,15 @@
   "^/doc/user-guide/dvc-internals(/.*)?$                                                  /doc/user-guide/project-structure/internal-files$1",
   "^/doc/user-guide/dvcignore$                                                            /doc/user-guide/project-structure/dvcignore-files",
   "^/doc/user-guide/contributing(/.*)?$                                                   /doc/contributing$1",
+  "^/doc/user-guide/contributing(/.*)?$                                                   /doc/contributing$1",
+  "^/doc/user-guide/large-dataset-optimization$                                           /doc/user-guide/data-management/large-dataset-optimization",
+  "^/doc/user-guide/external-dependencies$                                                /doc/user-guide/data-management/importing-external-data",
+  "^/doc/user-guide/pipelines/external-dependencies$                                      /doc/user-guide/data-management/importing-external-data",
+  "^/doc/user-guide/managing-external-data$                                               /doc/user-guide/data-management/managing-external-data",
+  "^/doc/user-guide/pipelines/external-outputs$                                           /doc/user-guide/data-management/managing-external-data",
```

These redirects are a little weird. Even if we want to duplicate page entries like that, the navigation via redirect can be confusing (it moves you unexpectedly in the nav).

> they are about external data but only really make sense in the context of pipelines

I'd just put them under pipelines for now and link from certain data mgmt pages as appropriate.

---

Let's at least make them 302s for now?
```suggestion
  "^/doc/user-guide/external-dependencies$                                                /doc/user-guide/data-management/importing-external-data",
  "^/doc/user-guide/pipelines/external-dependencies$                                      /doc/user-guide/data-management/importing-external-data 302",
  "^/doc/user-guide/managing-external-data$                                               /doc/user-guide/data-management/managing-external-data",
  "^/doc/user-guide/pipelines/external-outputs$                                           /doc/user-guide/data-management/managing-external-data 302",
```

---

> I'd just put them under pipelines for now and link from certain data mgmt pages as appropriate.

I think I'd rather leave them in data management and remove from pipelines here -- we already link to external dependencies in https://dvc.org/doc/user-guide/pipelines/defining-pipelines#simple-dependencies.

---

Do use 302s for now though, please!

---

Done, thanks!